src/lib.rs

```diff
@@ -14,7 +14,7 @@
 //!  - Live watching and re-reading of configuration files
 //!  - Deep access into the merged configuration via a path syntax
 //!  - Deserialization via `serde` of the configuration or any subset defined via a path
-//!
+//! Please note that this crate is case insensitive and all the keys are converted to lowercase internally.
```

Comment:
I guess you can leave the blank line after the listing. AFAIK having a blank line after a listing is considered good style in markdown.

Care to change this? :laughing:

Reply:
Oh okay, interesting. Can you link me to the CI failure for this?

Reply:
Yes, I think is this [one](https://github.com/mehcode/config-rs/actions/runs/2615343121)

Reply:
Ah. Yes, I see what gave you that impression: If you look closely, you'll see that on line 11 of the log you linked, the line is removed, but on line 12 it is added again.

This is usually an indicator of trailing whitespace that was removed! So there was a trailing whitespace that got removed, which shows as the line removed (line 11) and it looks like the very same line was added (line 12), but it is one without a trailing whitespace.
Unfortunately it is not clearly visible in the log.

I don't mind this, so I will mark this as resolved. :laughing: